tests/subscriptions/test_codecs.py

```diff
@@ -84,10 +110,17 @@ def test_encode() -> None:
     assert data["time_window"] == int(subscription.time_window.total_seconds())
     assert data["resolution"] == int(subscription.resolution.total_seconds())
 
+    if organization:
+        assert data["organization"] == subscription.organization
```

Comment:
You should be able to assert this regardless of `organization`, since both values should be `None`.